learning/func.py

```diff
@@ -0,0 +1,179 @@
+import io
+import os
+import json
+import logging
+import aiohttp
+import asyncio
+import feedparser
+import oci
+import datetime
+import uuid
+import fdk
+
+from fdk import response
+from slack import WebClient
+from slack.errors import SlackApiError
+
+local_dev = os.environ.get('is_local')
+slack_bot_token = os.environ.get('slack_bot_token')
+compartment_id = os.environ.get('compartment_id')
+auth_token = os.environ.get('web_auth_token')
+
+TABLE_NAME = "learning_posts"
+
+logger = logging.getLogger()
+slack_client = WebClient(token=slack_bot_token, run_async=True)
+
+
+# TODO: feed in via env config or db
+learning_sites_rss = [
+    {
```

Comment:
Why isn't this in a flat file of its own instead of in-source? Unless there is a problem with shipping multi-file functions, I'd like to have configuration independent of logic.

Reply:
great thought, actually the next step is to move this to a DB to make it more dynamic (See https://github.com/bluegrass-devs/Gilda/issues/3)